mapper/address.go

```diff
@@ -0,0 +1,14 @@
+package mapper
+
+import (
+	ethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+// ConvertHashToAddress uses the last 20 bytes of a common.Hash to create a common.Address
+func ConvertHashToAddress(hash *ethcommon.Hash) *ethcommon.Address {
+	if hash == nil {
+		return nil
+	}
+	address := ethcommon.BytesToAddress(hash[12:32])
```

Comment:
Do you just accept panic here? or can you return an error?

Reply:
Shouldn't be an error or panic, ethcommon.Hash is always 32 bytes long

Reply:
If it changes without us knowing, I'm fine with the panic